libtracepoint-control-cpp/include/tracepoint/TracepointSession.h

```diff
@@ -242,6 +267,9 @@ namespace tracepoint_control
         tracepoint_decode::PerfEventMetadata const&
         Metadata() const noexcept;
 
+        tracepoint_decode::PerfEventDesc const&
+        EventDesc() const noexcept;
+
```

Comment:
Note: Add support for getting full PerfEventDesc info about each tracepoint that has been added to the session.